src/routes/+error.svelte

```diff
@@ -125,6 +126,7 @@
 </script>
 
 {#if matrix}
+  <Seo title="4🥚4" />
```

Comment:
doesn't need to be conditionally rendered depending on `matrix`.
Please move out of the if statement